### PR TITLE
add documentation for Operational Validation SIG

### PR DESCRIPTION
**Pull request by skandragon**
This is a proposal for an operational validation SIG.  See the commits for details.

### CODE REVIEW

**Comment by aglover** [comment]
+1 binding vote

**Comment by duftler** [comment]
+1

**Comment by duftler** [comment]
This proposal is ready for a vote.

@demobox, @pstout, @rmeshenberg

File: sig-operational-validation/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* Establish a common framework for evaluating and improving validation techniques.
+* Facilitate community contributions to the Spinnaker ecosystem where validation methods can be inserted into pipelines and other
+decision points.
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
+
+**[Michael Graff](https://github.com/skandragon)**
+Principal Architech at OpxMX
```
Comment (pstout) [inline sig-operational-validation/README.md]: Typo or creative job title: Architech versus Architect

**Comment by demobox** [comment]
+1

**Comment by skandragon** [comment]
I'm not clear on the next steps.  When and how is a vote performed?  It looks like things are approved (modulo the small typos, which I will fix if they are not already completed)

File: sig-operational-validation/README.md (hunk 1 above)
Comment (skandragon) [reply sig-operational-validation/README.md]: Fixed.

[the remark this answers is not in the document]
Comment (skandragon) [reply sig-operational-validation/README.md]: Fixed.

**Comment by aglover** [comment]
@skandragon OpsMx still appears misspelled?

**Comment by skandragon** [comment]
OK, so I'm still full of typos.  Heh.  Fix incoming.

**Pull request merged**